drivers/bridge/bridge.go

```diff
@@ -876,6 +877,54 @@ func setHairpinMode(nlh *netlink.Handle, link netlink.Link, enable bool) error {
 	return nil
 }
 
+func waitForIfaces(ifaceNames []string, timeout time.Duration, trigger func() error) error {
+	// Map interface names to their readiness states (false by default)
+	ifaceStates := make(map[string]bool)
+	allRunning := func() bool {
+		for _, ifaceState := range ifaceStates {
+			if !ifaceState {
+				return false
+			}
+		}
+		return true
+	}
+	for _, ifaceName := range ifaceNames {
+		ifaceStates[ifaceName] = false
+	}
+
+	// Start watching for link changes
+	notifyLink := make(chan netlink.LinkUpdate)
+	notifyDone := make(chan struct{})
+	netlink.LinkSubscribe(notifyLink, notifyDone)
```

Comment:
Sorry, the update I just pushed was not related to what you suggested.  (Your reply was within seconds of my `git push`.)  I will try your suggestion next.